### DISCUSSION TITLE
Loan Offer Selection

### DISCUSSION THREAD

**Discussion by SanjanaIL** [ONDC-Official/ONDC-FIS-Specifications#42] [General]
![image](https://github.com/ONDC-Official/ONDC-FIS-Specifications/assets/142986416/abb61f01-3be6-410f-b1d1-d667b4002ed6)

The borrower will see multiple offers from a single lender or offers from multiple lenders?
What all details will be available in the loan offer for the customer to evaluate?

**Comment by 92shreyansh** [comment]
@Antriksh-Parmar Please clarify the 1st question is this in scope of current MVP?
The details of the loans are available as part of your on_select response which the buyer app can identify how to provide evaluation UX to end customer

**Reply by Antriksh-Parmar** [reply]
one offer from a single lender.
